homeassistant/components/lutron_caseta/device_trigger.py

```diff
@@ -429,9 +456,9 @@ async def async_get_triggers(
 
 def _device_model_to_type(device_registry_model: str) -> str:
     """Convert a lutron_caseta device registry entry model to type."""
-    model, p_device_type = device_registry_model.split(" ")
+    *model_list, p_device_type = device_registry_model.split(" ")
```

Comment:
Maybe split this into a list, and then pop the list key out so we don't have to do the `*model_list`?

Reply:
I use this model_list in the next line to re-form the model.  
Is it preferred not to use the * notation?

Reply:
> Is it preferred not to use the * notation?

While, its is perfectly fine to use it, I generally prefer to avoid if there is another option.

I'd probably write it as:
```diff
-    *model_list, p_device_type = device_registry_model.split(" ")
-    device_type = p_device_type.replace("(", "").replace(")", "")
+    model_list = device_registry_model.split(" ")
+    device_type = model_list.pop().replace("(", "").replace(")", "")
```

Reply:
ok, I like that.  Sounds good.  I'll adjust.

Reply:
I've updated using your suggestion